### ISSUE TITLE
Library appears to have issues with maven classifiers

### ISSUE THREAD

**Issue by JohnnyJayJay** [pmonks/lice-comb#20]
I was using `lein-licenses` with [instant-poll](https://github.com/JohnnyJayJay/instant-poll) and it failed to detect the license of a maven artifact with a special classifier.

* `com.github.jnr/jffi$native []`

[`com.github.jnr/jffi`](https://mvnrepository.com/artifact/com.github.jnr/jffi/1.3.11) does have a declared license, the problem is (I'm assuming) lack of handling maven classifiers like `native`. Classified artifacts live in the same place as the default jar, and – as far as I'm aware – the license of the overall module applies to all artifacts.

**Comment by pmonks** [comment]
Thanks!  Do you happen to know which of `instant-poll`'s top-level dependencies ultimately reaches `com.github.jnr/jffi`?  That'll help with reproduction / unit tests.

**Comment by JohnnyJayJay** [comment]
It's [datalevin](https://github.com/juji-io/datalevin).
```
$ lein deps :why com.github.jnr/jffi
 [datalevin 0.5.27]
   [org.lmdbjava/lmdbjava 0.8.2]
     [com.github.jnr/jnr-ffi 2.2.2]
       [com.github.jnr/jffi 1.3.1]
```

**Comment by pmonks** [comment]
@JohnnyJayJay apologies for the delay in releasing this. I was going to cut a new release with this and issue #3 fixed, but the latter is taking longer than I expected.

Would you like me to cut an interim release with just this in it in the meantime?

**Comment by JohnnyJayJay** [comment]
> @JohnnyJayJay apologies for the delay in releasing this. I was going to cut a new release with this and issue #3 fixed, but the latter is taking longer than I expected.
> 
> Would you like me to cut an interim release with just this in it in the meantime?

I don't really care. I suppose it would be nice to have as a patch though.

**Comment by pmonks** [comment]
Released in v1.0.120.  Thanks again for reporting this!
